smallfile-wrapper/publisher.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+
+from __future__ import print_function
```

Comment:
Just curious of the future function here. What's the reason behind the need for the future function here? trying to figure out which print() call in the script required the need for it.  or is the assumption here that the interpreter is python2 ?

Reply:
It is because of the assumption that, by default, python2 interpreter will be invoked, when executing the script.